src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/BlockSectionProcessor.scala

```diff
@@ -0,0 +1,32 @@
+package org.ergoplatform.nodeView.history.storage.modifierprocessors
+
+import org.ergoplatform.modifiers.{BlockSection, ErgoPersistentModifier}
+import scorex.core.consensus.History.ProgressInfo
+import scorex.core.utils.ScorexEncoding
+
+import scala.util.Try
+
+/**
+  * Trait, that declares interfaces for validation and processing of various
+  * block sections: BlockTranasction, ADProofs, etc.
```

Comment:
Trait that declares interfaces for validation and processing of various block sections: BlockTransactions, ADProofs, etc.